.github/workflows/push.yml

```diff
@@ -115,3 +115,8 @@ jobs:
         run: |
           go run main.go bundle schema > schema.json
           ajv -s schema.json -d ./bundle/tests/basic/databricks.yml
+
+          # Valid configuration should pass schema validation.
+          for file in ./bundle/tests/jsonschema/allowed/*.yml; do
```

Comment:
Everything under `bundle/tests` has accompanying Go tests. I'd like to keep it that way so that you know which tests may be affected by changes. If someone reuses these YAML files for tests it introduces coupling.

Could you move this under `bundle/schema/testdata`?